packages/navigation/src/NavigationMenuDropdown.tsx

```diff
@@ -14,31 +15,52 @@ type NavigationMenuDropdownProps<T extends object> = CollectionBase<T> & {
 
 const NavigationMenuDropdown = <T extends object>(props: NavigationMenuDropdownProps<T>) => {
   const state = useListState(props);
```

Comment:
This is approach seems necessary because:

* There doesn't seem to be a way to configure a dropdown to set each menu item to the dropdown button's width
* The dropdown button gets cloned by dropdown before rendering as a popover, so we can't set a ref on `DropdownButton` from the outside

Reply:
Is this related to the collapse mechanism?

Reply:
Ignore me. We handled it in chat.